docs/index.md

```diff
@@ -0,0 +1,17 @@
+# ansible-compat
+
+```{eval-rst}
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+.. automodule:: ansible_compat.runtime
+.. autoclass:: Runtime
```

Comment:
Try this https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-option-automodule-special-members